### PR TITLE
Replace boolean min/max with bool_and/bool_or

### PR DESCRIPTION
**Pull request by electrum**
Also, add standard aggregation function 'every'.

### CODE REVIEW

File: presto-docs/src/main/sphinx/functions/aggregate.rst [hunk 1]
```diff
@@ -34,6 +34,10 @@ General Aggregate Functions
     Returns the number of ``TRUE`` input values.
     This function is equivalent to ``count(CASE WHEN x THEN 1 END)``.
 
+.. function:: every(x) -> boolean
```
Comment (martint) [inline presto-docs/src/main/sphinx/functions/aggregate.rst]: The argument should be boolean

File: presto-main/src/main/java/com/facebook/presto/operator/aggregation/BooleanMinAggregation.java [hunk 2]
```diff
@@ -21,7 +21,7 @@
 import static com.facebook.presto.operator.aggregation.state.TriStateBooleanState.NULL_VALUE;
 import static com.facebook.presto.operator.aggregation.state.TriStateBooleanState.TRUE_VALUE;
 
-@AggregationFunction("min")
+@AggregationFunction(value = "min", alias = "every")
```
Comment (martint) [inline presto-main/src/main/java/com/facebook/presto/operator/aggregation/BooleanMinAggregation.java]: Hmm... This relies on booleans being orderable and in a particular way (which I don't thing should actually be allowed). I'd prefer to have an explicit implementation of this function

Comment (electrum) [inline presto-main/src/main/java/com/facebook/presto/operator/aggregation/BooleanMinAggregation.java]: The min/max for Boolean are defined this way. We could delete them and add the Postgres bool_or function to replace max.

Does that sound reasonable?

**Comment by martint** [comment]
Yup, that sounds reasonable

**Comment by martint** [comment]
looks good

**Pull request merged**